abp.py

```diff
@@ -322,26 +327,32 @@ def crop_border(image):
         dpi = '{0:.2f}'.format(min(inner_card_width / 2.24, inner_card_height / 3.24))
         sheet_name = 'Sheet{:02d}_{}dpi.png'.format(sheet_index + 1, dpi)
         sheet_filename = os.path.join(output_dir, sheet_name)
-        sheet_image = Image.new('RGB', (sheet_width, sheet_height), 'white')
+        sheet_image = Image.new('RGB', (int(sheet_width), int(sheet_height)), 'white')
 
         for (i, (image, image_file, card, filename)) in enumerate(cropped_images):
+            print 'width:', image.width, 'vs', inner_card_width
```

Comment:
Yeah, forgot to undo debug prints

On Fri, Sep 9, 2016 at 1:05 PM Rob Ruana <notifications@github.com> wrote:

> In abp.py <https://github.com/RobRuana/abp/pull/1#discussion_r78214115>:
>
> >
> >          for (i, (image, image_file, card, filename)) in enumerate(cropped_images):
> > +            print 'width:', image.width, 'vs', inner_card_width
>
> Can you remove these bare print statements?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/RobRuana/abp/pull/1/files/24b6aa1334667b56eb921756c5c5d76fadb3cd85#r78214115>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAXMQtKa6bmB6xm3yWJvalBf3etWHr5eks5qoZGXgaJpZM4J5QG_>
> .
>